src/main/java/com/doongji/nestalk/service/user/UserService.java

```diff
@@ -59,4 +59,15 @@ public String findEmailByNameAndPhone(String name, String phone) {
                 .map(User::getEmail)
                 .orElseThrow(() -> new NotFoundException(User.class, name, phone));
     }
+
+    @Transactional
+    public User userDetails(Long userId) {
+        User findUser = userRepository.findById(userId).orElseThrow(() -> new NotFoundException("Not Found User"));
+        return findUser;
```

Comment:
```suggestion
        return userRepository.findById(userId).orElseThrow(() -> new NotFoundException("Not Found User"));
```
한줄로 깔끔하게 하면 보기 좋을것 같습니다 ㅎㅎㅎ

Reply:
리뷰 감사합니다 수정완료 하였습니다